tests/cases/fourslash/quickInfoJsDocTags7.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="fourslash.ts" />
+
+// @noEmit: true
+// @allowJs: true
+
+// @Filename: quickInfoJsDocTags7.ts
+////abstract class BaseClass {
+////    /**
+////     * Useful description always applicable
+////     * 

```

Comment:
The test needs to show what happens when there's a `param` tag for `stuff`, and what happens when there's a `param` tag for a parameter whose name is the same between base/derived.